### DISCUSSION TITLE
E6E Mastery Shard Feedback

### DISCUSSION THREAD

**Discussion by CaptainAlkohol** [NillerMedDild/Enigmatica6#5048] [General]
Hello there, 

I thought after finishing E6E (v1.13), I would give a bit of feedback specifically oriented at the endgame system, the shards. There has been plenty of feedback on the early- midgame already and i feel like i can help more on this specific topic. Maybe this helps some who would like to know how the endgame looks like in a bit more detail, maybe some people who already finished the pack already would also like to chime in and give their opinion on them. And perhaps it is of some use for the pack developers in making the endgame even a bit more fun. And speaking of, HUGE props to all the developers of this pack, it has been a great experience overall and you can see at every corner how much work went into this. If you havent tried it, its great, you will have a ton of fun!

Now, before i start, i finished the pack in around 360 hours, but we all know its Minecraft and mileage will vary drastically. I rushed through a whole lot of the automation (which bit me in the ass later on) and i also stopped trying to make things look nice after around 100 hours and went for progress. Everyone will have a wildly different experience here, just for context. I will adress each shard in order of my completion. This aligns almost completely with my percieved difficulty of the shards, but again, depending on your playstyle and the types of automation you really went hard on, this might also be different in your playthrough. I also will rank each shard on a scale of 1-10, this ranking will be a mixture of difficulty, theme, automation challenges and overall satisfaction on completion of the shard. 

I will try to keep my suggestions on a level which makes sense to implement so far into the development of a pack, I might get a bit carried away on a few occasions, feel free to ignore everything not feasible ;) So enough talk, lets get to it!

![image](https://user-images.githubusercontent.com/108421532/176768989-69cde2b1-7d3e-4ae4-976b-8f1c9ab9dfb5.png)

**Automation Mastery (8/10):**

This is the first shard the pack wants you to make and its a very good one at that. The only resource you really need a lot of is iron and you will probably have most of the automations already in place. This introduced the Create autocrafting system in case you havent messed with that already (no real need to before this), and I personally had to setup the automated Wicked Altar, everything else was in place already. I feel like the difficulty of this shard could be tuned up a notch, while its a fun little introduction, it REALLY doesnt prepare you for whats to come, most others are quite a jump up imo. Maybe a DIY modular router kit with a bit more cost than which is already used in the shard. Maybe diamond or even netherite crafters instead of the iron ones. Maybe throw some ID stuff in there, or any combination of those. It is in a good place and feels good to complete if its the first one you tackle, but that satisfaction quickly fades once you look at the next ones you have to do. 

**Energistics Mastery (9/10):**

Now, I said I completed the shards in order of difficulty, but those next 5 shards are really comparable depending on you existing automation and when you decided to farm certain things (Gapples and Habaneros say hi). So I did Energistics first, just because i was more on the Tech-side of things. This one is a fantastic shard, probably my favourite. Just a lot of machinery, great theme with basically sending huge reactors over and the first time you really need antimatter. This also gives you a reason to build the Stellar Neutron Activator and the Industrial Deuterium Plant, and building those Masterful Machineries is one of my favourite things about this pack. A bit of a shame that there is no real incentive to DO fusion in this pack, but honestly if it was, this shard would almost be too easy because you would have everything already. Now, just one small thought I had when making this shard, why make me do only Mekanism fusion in this Energistics shard? To be honest, this would be a bit more challenging and also a bit more reflective of most peoples playthroughs if there were things like fissile fuel, a fission reactor and a few Powah reactors in there. Maybe in place of some of the turbine parts, they are pmuch only there to make it bulky and cost more, none of those provide any actual difficulty. But still a great shard how it is.

**Aura Mastery (5/10):**

Now this is the first one that gets a lower rating. I think this shard has two problems, A) it is a bit too easy in my opinion and B) the entire "difficulty" comes pmuch only from ritual of the forest automation. Which in general, i dont even have a problem with to be clear, as it is a challenging automation and satisfying once you get it working well. BUT it is not really a challenge material-wise and pmuch only a waiting game AND... twisted saplings. In the weeks i have spent on the Discord server, i have not really found anyone providing a perfect solution for this but many people reporting the same problem. Those trees grow in weird ass shapes sometimes and make your rituals not work. Nothing is more frustrating than coming back to your craft after to hours and to see it stopped after 5/400. And babysitting those trees isnt much fun either. I tried things like placing blocks in positions that would stop it from growing weird shapes in multiple ways, i couldnt get it to work properly. IF there is a good solution out there, I'm not aware of it, but seems like I'm not the only one who ran into that problem and couldnt figure it out. A simple fix would be just to use a different sapling for sorrow and fear tokens imo, would make this shard a whole lot less painful. I dont have a good solution how to make this shard more difficult since everything NA-related just boils down to a waiting game tbh. Maybe some Aura generators instead of all those rails, or a buttload of ingots of the sky, but this might just be more of a problem with the mod itself than the shard.

**Culinary Mastery (7/10):**

Surprisingly less painful than i thought! I decided to go with the antimatter route for the Gapples, which saved me a ton of time and materials probably, and if anyone is low on Habaneros, i can really recommend the Hydroponic Beds from Industrial Foregoing. One thing that reduces the rating of this shard for me is that basically everything is provided by a Sylph if you set it up early enough and that its just "more of that thing you did for the School Project". Could have used maybe two types of meals, the student one and then the "Professor" ones, which could have used some more intricate foods and a bit more automation than just a Create mixer and a cutting board. And then the shard just take 25 of each instead of 50 of one. But this is one of those suggestions that come probably a bit too late in the development process, so i get that it will probably be a bit too much work to implement. I also dont have a perfect solution to counter the Sylph problem, but it sadly makes this shard just a bit too easy. Setting up some actual farms would have been fun, as i never had to in this whole playthrough. 

**Engineering Mastery (8/10):**

This one feels kinda similar to the Automation one, just a bit more expensive material-wise, and surprisingly easier automation-wise. I think the only bottleneck for me on this shard was the amount of lube needed (of course very much my own fault), the rest boils down to ten thousands of steel and some basic automation. Theme-wise it feels a bit weaker than most other shards since its just Create, PNC and IE things thrown together and the nature of those mods in the pack makes it pretty impossible to make this a more challenging and engaging automation challenge. But as you can tell from the rating, i still enjoyed it and it was a satisfying shard to turn in. 

**Alchemy Mastery (3/10):**

Now to the two weakest shards in my rating. First off, Alchemy. I might be in the minority, but does anyone actually enjoy potion making or uses them in any capacity? Not from the conversations i saw and had on Discord. And then a whole shard basically themed around that... Now i want to be clear, this is thankfully not very difficult, time consuming or expensive. I set up a few Thermal breweries, Wicked Altar, Occultism and Blood Magic were automated already, so it was only a matter of coding in recipes more or less. Im just not a fan of the shard in general and probably the items used in it. They feel like someone looked for anything having any relation to alchemy or potions and throwing it in there. Gapples? We had those already in a shard. Nocturnal Powder? You need exponentially more of that for other shards already. Bandages? Did anyone ever use one of those? Same goes for Token of Undying Friendship, Ring of last Chance and Plunderers Glint. Most people maybe used one of those in their playthrough, if even at all and it feels a bit disconnected because i personally feel those shards do a really good job of giving you basically a "Best-Of" of what you had to do to get to that point. I get the developers maybe felt like there needed to be another magic shard but something centered around Bees or Eidolon or Occultism would have been a better fit in my humble opinion. And this is not really a suggestion, just a thing I missed in all those shards was a "Processing" shard or something around those lines. With a ton of IF, Thermal and Mekanism machinery and upgrades, this would probably have been the shard i would have replaced. Probably my least favourite, but maybe some people really enjoy it so who am I to judge :)

**Astronomy Mastery (4/10):**

This shard is just... weird. Before i begin with my problems with it, i immensely enjoyed the Observatory crafts. They were somehow very satisfying to me to come up with clever ideas how to do it and then to see that whole Create thing work its ways. So this was a bit of a saving grace for me, otherwise this would have been my lowest rated one. This shard just doesnt feel very Astral Sorcery-y? First, the Automated Iridescent Altar gets used more in other, non-AS shards. Then there are 10k marble thrown in for no apparent reason but to imitate the building of an altar, which is neither fun nor challenging but fits the theme i guess. Then 500 attuned crystals... Once again, a thing that you just let run in the background for like 50hrs unless you wanna stand there with a TIAB. Now we all know that Astral automation can be a pain and thats the exact reason the automatic version exists in this pack. But still, i would have liked to see some challenge other than the observatory in here. Cut down the amount of crystals needed to 1 per shard, but instead make it that they need to have for example 3 purity. That would make you engage with ID or some other automation and also more with AS itself. Maybe some Mantles of Stars, Collector Crystals. Make it that you can only complete it once you reach 40 personal starlight levels. SOMETHING that makes this feel like Astral Sorcery. I actually dont think i touched an altar or crystal or relay or starlight once for this whole shard, which is a shame. 

**Ritual Mastery (7/10):**

This was a fun one again, very well themed, a few tiny extra automation challenges, and a pretty high but well balanced material cost. My only gripe with this is that it takes a long while to craft all the slates and moonstones and master orbs, but i guess people can build a second set of blood altars if they feel like it, I did it with one. Part of that problem is also that everything on the Tech side of shards also requires a shitton of slates (if you wanna do the doubling at least) and not so much a problem of this shard in itself. So my only suggestion would be to reduce the amount of runes a bit, maybe crank up some other stuff instead. Automate your Corundum early people, dont do it like me! And thanks to whoever decided to not make astronomical amounts of demonic will part of this, I appreciate it. 

**Excavation Mastery (7/10):**

Now this one would probably be a 9/10 if not for the Pedestal Quarry Kits. Very expensive, very heavy on Tech automation, with a little magic for the mineshafts thrown in there, great shard! But oh boy, those Pedestal Kits. Please reduce the amount of those needed. First off, its again not really reflective of actual playthroughs, maybe some people use one or two of them, but the shard is so well themed and balanced and then those are thrown in there seemingly just to increase cost and annoyance. Is it necessary for this one part to cost thousands of mana pearls, glyphs, runes, tokens and source, if half of the other shards require that same thing already? In my opinion its there to serve no other purpose than add a bunch of time consuming magic to the shard. Consider reducing it to one pedestal kit or even remove those things entirely and throw in some Digital Miners or RFTools builders. Would be a better experience. 

**Botanical Mastery (8/10):**

We are getting to the tough ones now! If you havent had a wake-up call by now, this is it. I hope you have a really stable and fast Botania system running, or oh boy you wont like this one. This one is difficult, takes a lot of time, requires a really cool automation setup for the Mana Collectors and is all in all a great shard. If i could improve this at all, I would probably reduce the amount of Mana Collectors needed by 25% or 50%, I feel like the automation should be the challenge, not the thousands of runes you need for so many things already anyways. Also a bit annoying since you need to craft those in batches unless you wanna craft 1000s of T1, T2 and T3 runes aswell. But not much to add here.  

**Battle Mastery (8/10):**

Very good shard aswell, depending on your automation setup it can be harder or easier than the Botanical or Dimensional ones. I did this one really late, so I had all the antimatter to produce tons of Powah crystals, without that its probably a bit more painful to farm 1k nether stars and 10k+ of those crystals or however many you need. The Mek-Suit-Kits are hilarious, tons of weird new recipes, fluids, gases and materials you need to figure out how to get. This was the first time in like 100 hours i actually had to find some new things i believe, was a great change of pace. As you probably have guessed by now, no shard can be without a suggestion so for this one, the only thing that felt a little bit out of place were the teleporters. A bit random, not really adding a lot and even a bit more fitting to the Dimensional shard imo. Those things could be scrapped in place of some other high end weaponry or armour, would give a bit more variety than just a Mekanism shard too. 

**Dimensional Mastery (7/10):**

Oh boy. Now i finished this pack in the 1.13 version (which i hadnt thought i would just a week before) and I know the amount of disks needed for this gets halved in the next update. But holy hell, even then this will be the most difficult one by far. This shard requires exponentially more upscaling AND raw materials than any other shard and its not close. So if you are progressing the shards right now, code in the recipe for this and start mass producing. 10 Pressure Chambers and Advanced Assembly Tables wont hurt either. I started the production and crafts for this around my 2nd or 3rd shard and i still finished it last by a long shot. If you follow the Bragging Right quest line and look at its cost only when it leads you there, you will be up for a bad surprise. Be prepared to spend 100s of thousands of silicon, redstone, aluminum, andesite, plastic, sawdust, hardened glass and others. Robust Occultism automation is also very necessary. Get them Alloy Smelters, Multiservo Presses and Fluid Encapsulators running early. But enough advice, lets get some feedback in. Theme-wise, absolutely great shard, ticks all the boxes (although i would move the teleporters form the Battle shard here as mentioned). But i do think the cost is still too much in relation to other shards. I mentioned to make some a bit more expensive, this could be made cheaper to make the progression through the shards more of a curve and less of a rocky mountain. And this is pmuch only Disk-related, all the other items required are totally fine! I think making each shard cost 2 or maybe a max of 3 per disk would bring this in a much better place and let this still be the most expensive one, just not by such a ridiculous amount. 

And that is it, my feedback on the mastery shards! I hope at least some people made it through my ramblings to the end, feel free to agree or disagree and discuss my suggestions and how you feel about the shards in general. I hope there was also at least some helpful feedback for the developers, again my huge thanks for making this pack so much fun. 

Now, I said I would keep my feedback only shard-related but there are 2 small suggestions that are not but still fit here:

1) The Engimatic Tree of Life. I had to reform the structure about 6 times (I'm gonna be honest, i went into creative for that) because SOMETHING broke. I honestly have no idea what, i suspect some grass or rope or leaf or mushroom or whatever else of a more fragile nature. I could never trace it down because the structure is just that huge, fixing it by hand is possible but a nightmare as you would need to keep excess of every single material somewhere. So my suggestion, keep the structure exactly as it is, just make the controller only check for the more sturdy building blocks to form a valid structure. Aka, the Wood, the cave and the ports. Nobody needs to know about this so nobody will feel the need to cheat the system (its not really that hard to build anyways, so why would you) but it would reduce the computational requirements for checking the structure AND make it not break so easily. No idea if other people had similar experiences, just my 2 cents.

2) A suggestion that comes from spending a lot of time on the Discord server and answering questions there: Give Sylphs and Drygmies a WAY more prominent spot in the quest book. They are close to required to beat this pack unless you want to build countless farms, both for mobs and for plants and kill way more Masticators and Warped Moscos than any sane person would want to. With the spot they currently have in the Ars questline, they are just so easy to miss, if not for the Discord I would have too. They are so important that they honestly could even have a spot in the Expert questline (Magic-Apprentice for Drygmy and Magic-Adept for Sylph maybe?). Plenty of people coming into the server asking for a specific resource and 99% of the answers are Drygmy and Sylph (with 99% of reactions being "a what now?"). Think this would help a lot of players in the future. 

Now this is really finally the end, thanks everyone for reading, thanks to the developers and thanks to the great Discord community, see you there and looking forward to E9E ;)

**Comment by MuteTiefling** [comment]
Awesome feedback, thanks :D 

You've given me a lot to consider here, but I've got a few ideas on how to make some of these pain points less painful. Will hopefully be able to get them coded up in the next week or so.

**Comment by NielsPilgaard** [comment]
Very nice feedback, it was a pleasure to read :D

**Comment by MuteTiefling** [comment]
Bringing this over here for my own reference:

> Wrote this on Discord but you probably missed it. Regarding the Alchemy shard: I think it would maybe be a fun idea to make it a "brewery" shard? Wouldnt need a change of the art. Then you could do things like Potion Brewers, Alchemical Imbuers, a IE Fermenter Kit and a Mixer Kit, and have it require a lot of some liquids like Ethanol and Potions. I think thats more fun than the bottled versions. Maybe keep a few of the original ones like the bandages so that the original purpose doesnt get completely lost and you still have some potion bottles in there. I think the traps are fine aswell. That way you could throw in some more Simple Farming Brewery as well, and i think it would be thematically a good counterpart to the meals

**Reply by MuteTiefling** [reply]
So, the intent here, at least according to the lore, was to give the communities essentially medical aid. Some machinery is good though, again since they're meant to be becoming self sufficient. 

So an IE Mixer and Bottler kit are good, I think. 
Also a stim pack with multiple items in it, rather than a stack of bandages. This would be a sequential craft containing things to quickly get someone back on their feet, as well as to keep them safe. 

Things like the soul gem and BM anointment don't fit so well, so those'll go. And nocturnal powder is absolutely everywhere in other crafts, so it's gone. 

I think that'll tidy it up a bit. Yes there'll be fewer actual items, like in the culinary, but a more cohesive theme.

**Reply by SoulfieUwU** [reply]
Do you think it's possible to use outputs from the Potion Melder from Ars Noveau? I think it'd be pretty neat for players to mix some a variety of potion effects for the shard.

**Reply by MuteTiefling** [reply]
It should be feasable, but I fear it may be too confusing to mix the exact right potions. Would be a nice fit as a replacement for the planned botania potions though, so I'll give it some thought.
